9-publish to PyPi (#15)
NetherlandsForensicInstitute/adb-pywrapper: CONTRIBUTING.md

Reviewer: adding this file closes #3

## Pull Requests

- All contributions to the project must be submitted via pull request.
- Ensure that your pull request addresses a specific issue or feature request. If none exists, please open a new issue first to discuss the changes you'd like to make.
- Follow the [GitHub Flow](https://guides.github.com/introduction/flow/) workflow:
  1. Create a new branch from `main`. The branch name should start with the issue number.
  2. Make your changes and commit them with descriptive commit messages.
  3. Submit a pull request to the main repository's `main` branch.
- Ensure that your code adheres to the project's coding standards and conventions.
- Include unit tests if applicable and ensure that existing tests pass.
- Provide a clear and detailed description of your changes in the pull request description.

## Issues

- Feel free to open an issue for bug reports, feature requests, or general feedback.
- When opening an issue, please provide as much detail as possible, including steps to reproduce for bug reports.

## License

By contributing to this project, you agree that your contributions will be licensed under the project's [LICENSE](LICENSE.txt).

> ## Releasing

Reviewer: releasing is not part of contributing

Author: Yes good point, what is a better place for this information. I wanted to have it somewhere

Reviewer: I would make a RELEASE_GUIDELINES.md

- Creating a release in GitHub automatically triggers publication on PyPI. Please ensure that your changes are properly documented in the release notes.

> - Semantic versioning should be used

Reviewer 2: Nitpick:
```suggestion
- Semantic versioning should be used.
```

> - The tag should be of the form v{version}.

Reviewer 2: ```suggestion
- The tag should adhere to the following format: `v{version}`
```
